Dockerfile.processing_base

```diff
@@ -20,5 +20,8 @@ RUN apt-get update && apt-get -yq install \
 RUN R -e "install.packages('BiocManager')"
 RUN R -e "BiocManager::install('SingleCellExperiment')"
 RUN R -e "install.packages(c('devtools', 'igraph'));"
+# note: Loom format has been deprecated from cellxgene but cellgeni/sceasy has a dependency on LoomExperiment
+#       See https://github.com/cellgeni/sceasy#installation
+RUN R -e "BiocManager::install('LoomExperiment')"
```

Comment:
This has nothing to do with tombstoning of collections, but this PR's build was broken by removing this dependency in #1465 , so re-adding it here.

Reply:
... and now, the bioconductor.org site appears to be down causing dependency installation to fail: 🤦 
```
...
#8 490.1   cannot open URL 'https://bioconductor.org/packages/3.12/bioc/src/contrib/LoomExperiment_1.8.0.tar.gz': HTTP status was '504 Gateway Timeout'
...
```